Sources/MapboxCoreNavigation/Directions+RoutingProvider.swift

```diff
@@ -17,7 +17,7 @@ extension Directions: RoutingProvider {
         return calculate(options, completionHandler: completionHandler)
     }
     
-    @discardableResult public func refreshRoute(indexedRouteResponse: IndexedRouteResponse, fromLegAtIndex: UInt32, completionHandler: @escaping RouteCompletionHandler) -> NavigationProviderRequest? {
+    @discardableResult public func refreshRoute(indexedRouteResponse: IndexedRouteResponse, fromLegAtIndex: UInt32, currentRouteShapeIndex: Int?, completionHandler: @escaping RouteCompletionHandler) -> NavigationProviderRequest? {
```

Comment:
We’ll need to add a property representing the `current_route_geometry_index` parameter to MapboxDirections so we can plumb this parameter down further.